Atomic/scan.py

```diff
@@ -34,32 +36,61 @@ def get_scan_info(scanner, scan_type):
                     for x in i['scans']:
                         if x['name'] == scan_type:
                             return i['image_name'], x['args'], i.get('custom_args')
+
+        def set_default_scanner():
+            # If no scanners are defined
+            if len(self.scanners) < 1:
+                count = 1
+                raise ValueError("No scanners are configured for your system.")
+            elif len(self.scanners) == 1:
+                self.default_scanner = self.scanners[0]['scanner_name']
+            else:
+                self.default_scanner = self.atomic_config['default_scanner']
+
```

Comment:
Do we need an error check to make sure default_scanner is in the self.scanners here?